backend-external/src/v1/routes/pay-transparency-routes.ts

```diff
@@ -166,7 +166,12 @@ const validateApiKey =
 router.get(
   '/',
   validateApiKey(config.get('server:apiKey')),
+  query('startDate')
+    .isISO8601()
+    .withMessage('Invalid start date format'),
+  query('endDate').isISO8601().withMessage('Invalid end date format'),
   utils.asyncHandler(async (req: Request, res: Response) => {
+    console.log(req.query)
```

Comment:
Was this just for debugging? If not, maybe it would be better to use the logger.ts